index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="intro">
+        <header id="header">
+            <h3 id="logo" class="content-padding"><a href="#intro">My Logo</a></h3>
+            <nav id="navbar">
+                <ul>
+                    <li class="content-padding"><a href="#">Portfolio</a></li>
+                    <li class="content-padding"><a href="#">About</a></li>
+                    <li class="content-padding"><a href="#">Contact</a></li>
+                </ul>
+                <a href="#" class="content-padding" id="menu"><img src="images/menu-icon.png" alt="menu"></a>
```

Comment:
Please fix the path to this image as it does not show.
<img width="560" alt="Screenshot 2022-09-23 at 12 34 11" src="https://user-images.githubusercontent.com/48546992/191932847-e5e692d9-8ee1-4a12-a851-106b8deca040.png">

Reply:
thanks sir I should share with you the changes I've made before submitting